onnxscript/tests/function_libs/torch_lib/ops_test.py

```diff
@@ -55,14 +55,23 @@ def _should_skip_test_sample(op_name: str, sample) -> Optional[str]:
     return None
 
 
+def _split_function_and_wrangler(
+    onnx_function_and_wrangler: Callable[..., Any]
+    | tuple[Callable[..., Any], Callable[..., Any]]
+) -> tuple[Callable[..., Any], Callable[..., Any] | None]:
+    """Splits a function with an optional input wrangler into a function and an input wrangler."""
+    if isinstance(onnx_function_and_wrangler, tuple):
+        return onnx_function_and_wrangler
+
+    assert callable(onnx_function_and_wrangler)
+    return onnx_function_and_wrangler, None
+
+
 class TestFunctionValidity(unittest.TestCase):
     def test_all_script_functions_are_onnx_functions(self):
         functions = set()
         for func_with_wrangler in ops_test_data.OPINFO_FUNCTION_MAPPING_SCRIPTED.values():
```

Comment:
nit: any consideration of making a `FunctionWithWrangler` type?

Reply:
Sounds like something I can try, but I will explore this after this whole stack is merged if that sounds good to you